### DISCUSSION TITLE
Time delay in python

### DISCUSSION THREAD

**Discussion by nandhakumarbalu** [libigl/libigl#1773] [Q&A]
HI

Can anyone help me  to provide a Python code for  creating a one minute time delay without sleep() function.

**Comment by S00PH11A** [comment]
Hi @nandhakumarbalu I saw a thread similar to your question on Stack Overflow.
It's quite old so I hope it's still useful.

[Check it out](https://stackoverflow.com/questions/15167334/alternative-to-pythons-time-sleep) to see if it still helps you.
